plugins/woocommerce-admin/client/products/product-title.tsx

```diff
@@ -48,6 +48,7 @@ export const ProductTitle: React.FC = () => {
 					? getProductVariation( {
 							id: parseInt( variationId, 10 ),
 							product_id: parseInt( productId, 10 ),
+							context: 'edit',
```

Comment:
Just thinking out loud: I wonder if `edit` should be the default context or not.  I'm guessing it should probably remain `view` since that's the default for most APIs on the server-side.

Reply:
Yes, since our datastore is almost always for edition I agree with you.